src/tools/perf/perftest_daemon.c

```diff
@@ -347,15 +348,48 @@ ucp_perf_daemon_send_handler(void *arg, const void *header,
     return UCS_OK;
 }
 
+static void ucp_perf_daemon_cleanup_connections(ucp_perf_daemon_context_t *ctx)
```

Comment:
instead of adding new function "ucp_perf_daemon_cleanup_connections", can we mov this code directly to fin_handler, so it will be symmetric to init_handler?

Reply:
It was implemented almost like you said, before this change. All the cleanup code was in `ucp_perf_daemon_cleanup`. Now the idea was to support `keep_running` mode, so that host process may connect to the daemon multiple times. This means that disconnection step is now separate from the overall cleanup, and now is encapsulated in `ucp_perf_daemon_cleanup_connections` function.
This function is called from 2 places: from FIN message handler, and from the main cleanup. So I cannot keep it inside the FIN handler

Reply:
ok